cfme/services/catalogs/orchestration_template.py

```diff
@@ -38,9 +38,13 @@ class CopyTemplateForm(ServicesCatalogView):
     cancel_button = Button('Cancel')
 
 
-class TemplateForm(CopyTemplateForm):
+class TemplateForm(ServicesCatalogView):
     title = Text('#explorer_title_text')
     template_type = BootstrapSelect("type")
+    name = Input(name='name')
```

Comment:
we need strict order of form filling. after we switch template_type - UI resets content

Reply:
would you mind adding appropriate comment ?